src/HospitalLibrary/Doctors/DoctorRepository.cs

```diff
@@ -15,5 +15,10 @@ public IEnumerable<Doctor> GetAllDoctorsWithSpecialityExceptId(SpecialtyType spe
         {
             return _dataContext.Doctors.Where(doctor => doctor.SpecialtyType.Equals(specialtyType) && doctor.Id != doctorId).ToList();
         }
+        public async Task<IEnumerable<Doctor>> GetTwoIternalMedicineDoctorsAscendingByPatientNumber()
+        {
+            return await _dataContext.Doctors.Where(doctor => doctor.SpecialtyType.Equals(SpecialtyType.ITERNAL_MEDICINE)).
```

Comment:
This is not a mistake, but you can move every method call to a new line, just to be more readable.